test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/BookkeeperK8sService.java

```diff
@@ -97,7 +97,18 @@ public boolean isRunning() {
                             log.debug("Current instance counts : Bookkeeper {} Controller {} SegmentStore {}.", currentBookkeeperCount,
                                       currentControllerCount, currentSegmentStoreCount);
                             if (currentBookkeeperCount != newInstanceCount) {
-                                return deployPravegaUsingOperator(zkUri, currentControllerCount, currentSegmentStoreCount, newInstanceCount, properties)
+                                final Map<String, Object> bookieSpec = ImmutableMap.<String, Object>builder()
```

Comment:
Please create bookie spec in a separate method and call it here. Keeps code more readable.

Reply:
done